Article/Article.js

```diff
@@ -101,14 +110,58 @@ const data = [
 
   Hint: You will need to use createElement more than once here!
 
-  Your function should take either an object as it's one argument, or 5 separate arguments mapping to each peice of the data object above.
+  Your function should take either an object as it's one argument, or 5 separate arguments mapping to each piece of the data object above.
+*/
+
+
+
+const godFunction = function(title, date, content1, content2, content3){
```

Comment:
I know it is just a goof, but naming your functions consistent and helpful things is a very important, and hard, part of being a developer.